api/envoy/http/service_control/config.proto

```diff
@@ -126,4 +125,7 @@ message FilterConfig {
 
   // The Http uri to call service control
   api.envoy.http.common.HttpUri service_control_uri = 8;
+
+  // The prefix added to generated headers
+  string generated_header_prefix = 9 [(validate.rules).string.min_bytes = 1];
```

Comment:
Also add the http header value validation that other ones have. Make sure to include `strict: false`.

Reply:
Hi Teju, what is difference between `strict: true/false`? I haven't found doc about this.

Reply:
```
// x must conform to a well known regex for HTTP header names (via RFC 7230)
string x = 1 [(validate.rules).string.well_known_regex = HTTP_HEADER_NAME]

// x must conform to a well known regex for HTTP header values (via RFC 7230) 
string x = 1 [(validate.rules).string.well_known_regex = HTTP_HEADER_VALUE];

// x must conform to a well known regex for headers, disallowing \r\n\0 characters.
string x = 1 [(validate.rules).string {well_known_regex: HTTP_HEADER_VALUE, strict: false}];
```

Reply:
It essentially only validates it doesn't contain `\r\n\0` characters, because these will crash envoy at runtime.

Reply:
If strict is true, then it will disallow all characters that are not in RFC 7230. But envoy can handle some of these characters anyways